dog.go

```diff
@@ -39,9 +39,10 @@ const zombieSafeRadius float64 = 192
 // states of the dog
 // It would be great to map them to the frameTag.Name from JSON
 const (
-	dogWalking  = iota
-	dogSniffing
-	dogSitting
+	dogWalking  int = 0
+	dogFleeing      = 0
```

Comment:
Both Walking and Fleeing are zero. Is this a typo or is it a trick? Maybe the boundaries between logical game state and animation states have been blurred too much.

Reply:
Yes, exactly, but It has been changed since then and I rather did a mapping